v7/section_prevnext/section_prevnext.plugin

```diff
@@ -4,9 +4,10 @@ module = section_prevnext
 
 [Documentation]
 author = Alexander Krimm
-version = 0.8
-website = https://getnikola.com/
+version = 0.1
+website = https://plugins.getnikola.com/#section_prevnext
```

Comment:
that isn’t going to be the URL anymore. You can use your own homepage here, or whatever else.